pylxd/image.py

```diff
@@ -80,7 +81,66 @@ def create(cls, client, image_data, public=False, wait=False):
             Operation.wait_for_operation(client, response.json()['operation'])
         return cls(client, fingerprint=fingerprint)
 
+    @classmethod
+    def create_from_config(cls, client, config, wait=False):
```

Comment:
Ah, I see now. It's being called from the other method you're creating. A couple of things here:

- This doesn't need to be a method. It can be a function not contained scoped in `Image`.
- This shouldn't be a public function. Preface it with a `_`. We want to be able to modify it's usage without breaking people (and if people are using functions prefaced with `_`, I can't care about them).